function_examples/rest/README.md

```diff
@@ -1,2 +1,7 @@
-# Example function
-This function gets data from https://jsonplaceholder.typicode.com/, and also has a build process.
\ No newline at end of file
+# Rest
+
+This is a basic example of a function that when called, makes a request to a rest API using `node-fetch`, and returns a portion of the response back to the client.
+
+It includes an `index.html` that makes the call and displays the response.
+
+This function gets data from https://jsonplaceholder.typicode.com/, and also has a build process.
```

Comment:
I would say that netlify-lambda is what's being used for the build process, and that it takes the source files and package.json from `/func` and builds the function as a single file in a new folder, `/functions`. I would link to netlify-lambda for further information on local serving, build commands, etc.